docs/configuration.md

```diff
@@ -9,7 +9,7 @@ using [HOCON](https://github.com/typesafehub/config) syntax.
 Here is an example `.scalafmt.conf`:
 
 ```scala config
-align = true    // For pretty alignment.
+align = true    // For pretty alignment. (equivalent to align=more)
```

Comment:
Let's write `align = more` instead, the true/false variants are only kept for legacy reasons

Reply:
Sure.